### PR TITLE
Fix validation issue for service graduation year

### LINKED ISSUE #4302: validation error in serviceAcademyGraduationYear
`Form The property '#/serviceAcademyGraduationYear' did not have a minimum value of 1900, inclusively in schema 6d3ac55d-0966-5868-be2d-0f1191776ce9#`
happened on Dec 8, 2016 5:22:28 AM UTC

### PR DESCRIPTION
**Pull request by jbalboni**
Closes #4302.

Validation was letting invalid years through on serviceGraduationYear field.

### CODE REVIEW

File: src/js/edu-benefits/utils/validations.js [hunk 1]
```diff
@@ -268,7 +268,7 @@ function isValidTourOfDuty(tour) {
 }
 
 function isValidMilitaryServicePage(data) {
-  return !data.chapter33 || isNotBlank(data.benefitsRelinquished.value);
+  return isBlank(data.serviceAcademyGraduationYear.value) || isValidYear(data.serviceAcademyGraduationYear.value);
```
Comment (akainic) [inline src/js/edu-benefits/utils/validations.js]: The MilitaryServicePage is using `isValidCurrentorPastYear` for the validation on the `serviceAcademyGraduationYear` itself, so we should use the same validation here too so it's consistent.

**Review by akainic — COMMENTED** [review]
One small fix

**Comment by jbalboni** [comment]
@akainic Ready to be looked again

**Review by akainic — APPROVED** [review]
LGTM!

**Pull request merged**